Formula/zsh-git-prompt.rb

```diff
@@ -10,15 +10,6 @@ def install
     prefix.install Dir["*.{sh,py}"]
   end
 
-  def caveats; <<-EOS.undent
-    First, make sure zsh-git-prompt is loaded from your .zshrc:
-      source "#{opt_prefix}/zshrc.sh"
```

Comment:
Let's remove the "First" and then just reference the `source` and preceding line and skip the rest.